[2023-05-29T20:10:52.926+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_test_dag1.load_data manual__2023-05-29T20:10:46.661104+00:00 [queued]>
[2023-05-29T20:10:52.938+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_test_dag1.load_data manual__2023-05-29T20:10:46.661104+00:00 [queued]>
[2023-05-29T20:10:52.939+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-05-29T20:10:52.971+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_data> on 2023-05-29 20:10:46.661104+00:00
[2023-05-29T20:10:52.982+0000] {standard_task_runner.py:57} INFO - Started process 261 to run task
[2023-05-29T20:10:52.989+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'mysql_test_dag1', 'load_data', 'manual__2023-05-29T20:10:46.661104+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpzpx5pe7a']
[2023-05-29T20:10:52.992+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask load_data
[2023-05-29T20:10:53.092+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_test_dag1.load_data manual__2023-05-29T20:10:46.661104+00:00 [running]> on host 4fded2d8196b
[2023-05-29T20:10:53.221+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='mysql_test_dag1' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-29T20:10:46.661104+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-29T20:10:46.661104+00:00'
[2023-05-29T20:10:53.222+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-29T20:10:53.240+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=mysql_test_dag1, task_id=load_data, execution_date=20230529T201046, start_date=20230529T201052, end_date=20230529T201053
[2023-05-29T20:10:53.280+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-05-29T20:10:53.305+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
